README.md

```diff
@@ -47,11 +47,11 @@ For a more complete example around files operations, please check the [demo mobi
 
 ##### Directives
 
-| Name | Attributes |
-| ---- | ----------- |
-| wfm-img | uid |
-
-
+| Name | Attributes                   |
+| ---- | ---------------------------- |
+| wfm-img | uid                       |
+| file-detail | file, display-options |
```

Comment:
Probably needs an explanation of what `file, display-options` is with an example.